### DISCUSSION TITLE
Allow to react to / vote on / give thumbs-up on locked issues/threads

### DISCUSSION THREAD

**Discussion by keunes** [community/community#36107] [Projects and Issues]
Sometimes we lock discussions, because a decision has been reached within the team on how to proceed (how to implement a feature). To avoid noise via the infamous '+1' comments.

However, locking an issue prevents our community members to give a thumbs-up to the feature request. So in our attempt to create less noise we block any interaction and stop getting information that would be helpful for us.

Proposal
----
* When clicking 'lock conversation', ask the user if they want to 'still allow reactions?'. I suppose this can be done in a modal, with a checkbox 'Don't ask again'.
* Allow to change this setting later, through a 'Allow reactions' / 'Disable reactions' link in the right column, between 'Unlock conversation' and 'Pin issue'
* In the repository settings, add a setting that determines the default for issues within the repo.

**Comment by 2rs2ts** [comment]
This would resolve https://github.com/microsoft/vscode/issues/169665. I hope it gets implemented.

**Comment by DominiqueSite** [comment]
https://github.com/orgs/community/discussions/51556 is related.
Implementing this would also resolve https://github.com/AntennaPod/AntennaPod/issues/6557.

**Comment by kissu** [comment]
Almost 2026, AI is going crazy while basic features like this are still left on the curb. 
Priorities I guess. 🤷🏻‍♀️

**Reply by 2rs2ts** [reply]
@copilot, please write code to allow us to react to locked issues and threads
